src/Categories/Category/Instance/Properties/Setoids/Extensive.agda

```diff
@@ -46,9 +47,12 @@ Setoids-Extensive ℓ = record
            { to = λ z → conflict A B (eq {z})
            ; cong = λ {x} _ → conflict A B (eq {x})
            }
-        ; unique = λ {_} {_} {_} {_} {eq} _ _ {x} → conflict A B (eq {x})
         ; p₁∘universal≈h₁ = λ {_ _ _ eq x} → conflict A B (eq {x})
         ; p₂∘universal≈h₂ = λ {_ _ _ eq y} → conflict A B (eq {y})
+        ; unique-diagram = λ {X} {h} {i} eq₁ eq₂ {x} →
```

Comment:
The previous proof was quite a bit more succinct, any chance to re-capture that here too?

Reply:
Got it back to one line